sensors/SensorCommon.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+import numpy as np
+from gym import spaces
+
+
+def dist2circ(pointPos, circlePos, circleRadius):
+    diff = pointPos - circlePos
+    dist2center = (diff[0] ** 2 + diff[1] ** 2) ** (1 / 2)
```

Comment:
Simply use `np.linalg.norm(diff)`
It will avoid struggle when we want to move to higher orders, although it is unlikely to happen.

Reply:
done